spec/acceptance/nodesets/windows2016.yml

```diff
@@ -0,0 +1,19 @@
+---
+HOSTS:
+  m5pvtczysnfuhsw.delivery.puppetlabs.net:
```

Comment:
Was this meant to be committed?

Reply:
Nope thought I had removed that! There's also a few more changes that need to go in but I'll wait until you finish reviewing.